src/overlays/actors/ovl_En_Fg/z_en_fg.c

```diff
@@ -4,12 +4,38 @@
 
 #define THIS ((EnFg*)thisx)
 
+/*
+This may be a beta file of frogs, a version of them where they were all enemies
```

Comment:
I suggest using this format to add a description to an actor:
https://github.com/zeldaret/mm/blob/master/src/overlays/actors/ovl_Bg_Ctower_Gear/z_bg_ctower_gear.c